server/processors-alerts-publisher/src/main/java/org/hawkular/apm/processor/alerts/InvocationAlertsPublisherMDB.java

```diff
@@ -53,13 +55,41 @@
 
     @Override
     public void onMessage(Message message) {
-        logger.fragmentCompletionTimeReceived();
+        logger.invocationDetailsReceived();
         try {
             String data = ((TextMessage) message).getText();
-            List<CompletionTime> items = mapper.readValue(data, new TypeReference<List<CompletionTime>>() {});
-            publisher.publish(items, "FragmentCompletionTime");
+            List<NodeDetails> items = mapper.readValue(data, new TypeReference<List<NodeDetails>>() {});
+            List<Event> events = items.stream().filter(nd -> nd.isInitial())
```

Comment:
Perhaps the easiest way to deal with this, is that this PR is primarily about simplifying to remove the FragmentCompletionTime - which was a single 'metric' which is now being replaced essentially by service response time (derived from certain nodes that representing handling the server request). So for now suggest merging the PR, and then separately in discussions with the alerting team we work out whether it is better to publish more events?